docs/cudf/source/cudf_polars/memory_errors.md

```
@@ -0,0 +1,73 @@
+trade-off is that smaller chunks may reduce compute efficiency.
+
+```python
+engine = pl.GPUEngine(
+    executor="streaming",
+    executor_options={"target_partition_size": 64_000_000},  # 64 MB
+)
+```
+
+Or via environment variable:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)